webserver/pkgpkr/webservice/tests/test_selenium.py

```diff
@@ -100,9 +100,9 @@ def test_work_process(self):
         # Check if the user at my repositories page
         self.assertEqual(f'http://localhost:{LoginTest.port}/repositories', self.driver.current_url)
 
-        # The first element from the repos list
+        # The last element from the repos list
```

Comment:
I think it would be better to choose the repository by name, so the test isn't quite as brittle.

There should be examples elsewhere in the Selenium test that show how to select an element by the text it contains (we do this for the pagination element on the recommendations page, for example).